### PR TITLE
en: add what's new in v1.1

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/325
- Other reference link(s):

### CODE REVIEW

File: en/whats-new-in-v1.1.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: What's New in v1.1
+summary: Learn the new features in TiDB Operator v1.1.
+category: introduction
+---
+
+# What's New in v1.1
+
+Based on 1.0, TiDB Operator 1.1 adds many new features, including TiDB 4.0 support, TiKV data encryption, TLS certificate configuration. It also support deploying new components such as TiFlash and TiCDC.
```
Comment (TomShawn) [inline en/whats-new-in-v1.1.md:9]:
```suggestion
Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.
```

File: en/whats-new-in-v1.1.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+TiDB Operator 1.1 also makes improvements in usability, providing user experience consistent with the Kubernetes native resources.
```
Comment (TomShawn) [inline en/whats-new-in-v1.1.md:11]:
```suggestion
TiDB Operator 1.1 also makes improvements in usability, providing the user experience that is consistent with the Kubernetes native resources.
```

File: en/whats-new-in-v1.1.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Extensibility
+
+- TidbCluster CR supports deploying and managing the PD Discovery component, which is fully capable of replacing tidb-cluster chart to manage the TiDB cluster.
+
+- TidbCluster CR adds support for Pump, TiFlash, TiCDC, and TiDB Dashboard.
+
+- Add the [Admission Controller](enable-admission-webhook.md) (optional) to improve the experience of upgrade and scaling and to provide the gated release feature.
```
Comment (TomShawn) [inline en/whats-new-in-v1.1.md:19]:
```suggestion
- Add the [Admission Controller](enable-admission-webhook.md) (optional) to improve the user experience of upgrade and scaling, and to provide the gated release feature.
```

File: en/whats-new-in-v1.1.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+- Support using tikv-importer chart to [deploy and manage tikv-importer](restore-data-using-tidb-lightning.md#deploy-tikv-importer).
+
+## Usability
+
+- Add TidbMonitor CR to deploy the cluster monitoring.
+
+- Add TidbInitializer CR to initialize the cluster.
+
+- Add Backup, BackupSchedule, and Restore CR to back up and restore the cluster, which supports using Amazon S3 and GCS as remote storage.
```
Comment (TomShawn) [inline en/whats-new-in-v1.1.md:31]:
```suggestion
- Add Backup, BackupSchedule, and Restore CR to back up and restore the cluster, which supports using Amazon S3 or GCS as the remote storage.
```

File: en/whats-new-in-v1.1.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+## Security
+
+- Support configuring TLS certificates between TiDB components and for MySQL client.
+
+- Support TiKV data encryption.
+
+## Experimental features
+
+- Add `TidbClusterAutoScaler` to implement [automatic scaling](enable-tidb-cluster-auto-scaling.md). You can enable this feature by turning on the `AutoScaling` switch.
```
Comment (TomShawn) [inline en/whats-new-in-v1.1.md:43]:
```suggestion
- Add `TidbClusterAutoScaler` to implement [auto-scaling](enable-tidb-cluster-auto-scaling.md). You can enable this feature by turning on the `AutoScaling` switch.
```

**Comment by TomShawn** [comment]
LGTM
@cofyc PTAL

**Comment by ran-huang** [comment]
@cofyc PTAL

File: en/whats-new-in-v1.1.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+---
+title: What's New in v1.1
+summary: Learn the new features in TiDB Operator v1.1.
+category: introduction
+---
+
+# What's New in v1.1
+
+Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.
```
Comment (DanielZhangQD) [inline en/whats-new-in-v1.1.md:9]:
```suggestion
Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration, etc. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.
```

File: en/whats-new-in-v1.1.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+TiDB Operator 1.1 also makes improvements in usability, providing the user experience that is consistent with the Kubernetes native resources.
+
+## Extensibility
+
+- TidbCluster CR supports deploying and managing the PD Discovery component, which is fully capable of replacing tidb-cluster chart to manage the TiDB cluster.
+
+- TidbCluster CR adds support for Pump, TiFlash, TiCDC, and TiDB Dashboard.
+
+- Add the [Admission Controller](enable-admission-webhook.md) (optional) to improve the user experience of upgrade and scaling, and to provide the gated release feature.
```
Comment (DanielZhangQD) [inline en/whats-new-in-v1.1.md:19]:
```suggestion
- Add the [Admission Controller](enable-admission-webhook.md) (optional) to improve the user experience of upgrade and scaling, and to provide the canary release feature.
```

File: en/whats-new-in-v1.1.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+- Add TidbInitializer CR to initialize the cluster.
+
+- Add Backup, BackupSchedule, and Restore CR to back up and restore the cluster, which supports using Amazon S3 or GCS as the remote storage.
+
+- Support gracefully restart a component in the TiDB cluster.
+
+## Security
+
+- Support configuring TLS certificates between TiDB components and for MySQL client.
```
Comment (DanielZhangQD) [inline en/whats-new-in-v1.1.md:37]:
```suggestion
- Support configuring TLS certificates for the TiDB cluster components and for MySQL clients.
```

File: en/whats-new-in-v1.1.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+- Support TiKV data encryption.
+
+## Experimental features
+
+- Add `TidbClusterAutoScaler` to implement [auto-scaling](enable-tidb-cluster-auto-scaling.md). You can enable this feature by turning on the `AutoScaling` switch.
+
+- Add the optional [Advanced StatefulSet Controller](advanced-statefulset.md), which supports deleting a specified Pod. You can enable this feature by turning on the `AdvancedStatefulSet` switch.
```
Comment (DanielZhangQD) [inline en/whats-new-in-v1.1.md:45]:
```suggestion
- Add the optional [Advanced StatefulSet Controller](advanced-statefulset.md), which supports deleting a specific Pod. You can enable this feature by turning on the `AdvancedStatefulSet` switch.
```

File: en/whats-new-in-v1.1.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
+---
+title: What's New in v1.1
+summary: Learn the new features in TiDB Operator v1.1.
+category: introduction
+---
+
+# What's New in v1.1
+
+Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration, etc. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.
```
Comment (TomShawn) [inline en/whats-new-in-v1.1.md:9]: Latin abbreviation is not recommended in technical documentation. `Including` here means inexhaustive and the list can actually go on.
```suggestion
Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.
```

**Pull request merged**